README.md

```diff
@@ -1,9 +1,145 @@
-# Create Ignite App
+# Create Ignite App Template
 
-[NoA Ignite](https://noaignite.se/) App Scaffolding built with [React](https://reactjs.org/) and [MUI](https://mui.com/).
+This is the official [NoA Ignite](https://noaignite.se/) base template. It's built with [Next.js](https://nextjs.org/), [Storybook](https://storybook.js.org/), [MUI](https://mui.com/) and [NoA Ignite packages](https://www.npmjs.com/org/noaignite).
+If something doesn’t work, please [file an issue](https://github.com/noaignite/create-ignite-app/issues).
 
-## Getting started
+## Getting Started
 
-- Edit `name` & `author` in package.json
-- Generate favicons on [favicon.io](https://favicon.io/) and replace the files under /public/
-- Add font files to /public/fonts/ and import them under /src/components/internal/MuiCssBaseline.js and /src/pages/_document.js (We only preload woff2 as it's supported by all browser other than IE11, and adding the media-query wizardry for that doesn't feel worthy of 2020. Do note that the crossOrigin prop may need to change if you're using a different domain for CDN, or loading the font from somewhere else.)
+```sh
+npx @noaignite/create-app my-app
+cd my-app
+```
+
+### Developing with Storybook
+
+```sh
+yarn storybook
+```
+
+Then open http://localhost:3001/ to see your app in Storybook.
+If you need to deploy Storybook app to production, create a minified bundle with `yarn build-storybook`.
+
+### Developing with Next.js
+
+```sh
+yarn dev
+```
+
+Then open http://localhost:3000/ to see your app in Next.js.
+When you’re ready to deploy your Next.js app to production, create a minified bundle with `yarn build`.
+
+## Developing the App
+
+Inside the newly created directory, the initial project structure will look something like:
+
+```
+my-app
+├── .husky
+├── .storybook
+├── node_modules
+├── .eslintrc.js
+├── .prettierrc
+├── babel.config.js
+├── next.config.js
+├── package.json
+├── public
+│   ├── fonts
+│   ├── locales
+│   ├── media
+│   └── favicon.ico
+└── src
+    ├── api
+    ├── blocks
+    ├── components
+    ├── containers
+    ├── pages
+    ├── utils
+    └── pages.stories.js
+```
+
+The philosophy behind some of this structure can be read below.
+
+### src/api
+
+This is a good place to add App-level providers that should be accessible from all corners of the application. Providers added here should be exported in the api directory's common export file(s). Accessing data or helpers from these providers can look like this `import { useI18n } from 'api'`.
+
+```
+src/api
+├── __mock__ (Mock data used for Storybook)
+├── CheckoutContext (Checkout-API data & helpers)
+├── I18nContext (Internalization data & helpers)
+├── SettingsContext (CMS data)
```

Comment:
Maybe "SettingsContext (global settings and CMS data)" ?

Reply:
👍